pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -691,6 +691,9 @@ func (ctx kubevirtContext) Info(domainName string) (int, types.SwState, error) {
 		res, err = getVMIStatus(vmis, nodeName)
 	}
 	if err != nil {
+		if strings.Contains(err.Error(), "etcdserver: request timed out") || strings.Contains(err.Error(), "apiserver not ready") {
```

Comment:
I would suggest creating a function to avoid repeating this (and having only one place that needs to be updated if we find more error messages to include).
Something like: `func isK3sUnreachableError(err error) bool`

Reply:
Yeah that makes sense.